### ISSUE TITLE
401 unauthorized

### ISSUE THREAD

**Issue by choudhury009** [postaddictme/instagram-php-scraper#1102]
Hi, after a couple of successful tries of retrieving media using getMedias() for a public profile. It is now saying 401: Unauthorized.

Any ideas on what the issue is and how to overcome this. 

Thanks

**Comment by carlosanpardo** [comment]
Could this be an IP blocking issue?
- Have you tried to change the IP to make the query?

**Comment by choudhury009** [comment]
@carlosanpardo yes I’m thinking the same, it could be that. Do you know how to add proxies to a server? I’m currently using a Digital Ocean Ubuntu server.

**Comment by carlosanpardo** [comment]
I test from my laptop with XAMPP and works (without credentials). Using a Web server appear and 401 unauthorized. Any idea to use the proxy?
